### PR TITLE
fix_1274_remove_bash

### PR DESCRIPTION
**Pull request by clanmills**
See #1274.  Remove bash from the test suite.  Invoke version_test etc from python

### CODE REVIEW

**Comment by kmilos** [comment]
Confirmed `cmake --build . --target tests` runs ok on my local mingw build.

I'll just push a commit to fix indentation (tabs vs spaces).

**Review by kmilos — APPROVED** [review]

**Comment by clanmills** [comment]
Thank You for looking at this, Milos.  I've removed Leo's permissions to write in the repos.  I've given Leo about 100 hours of coaching and mentoring - that's enough.  I'm fixing things up in order to get the Exiv2 v0.27.4 RC1 finished.  The code in tests/bash_tests will have to stay for now.  It needs attention and cleaning up.

File: tests/bash_tests/unit_test.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+import os
+import unittest
+
+bSkip=not (os.path.isfile(system_tests.unit_tests) or os.path.isfile(system_tests.unit_tests + '.exe'))
```
Comment (kmilos) [inline tests/bash_tests/unit_test.py:7]: Could potentially use the platform agnostic [`shutil.which()`](https://docs.python.org/3.8/library/shutil.html#shutil.which) and not worry about extensions...

Comment (clanmills) [reply tests/bash_tests/unit_test.py:7]: I didn't think of that, @kmilos.  I don't know python very well.  This is good enough for now.  It's working a lot better now than it was a couple of hours ago.

When the CI goes green, it's time to merge.  I want to make progress with the release itself today.  It's not due until 2 weeks today.  Nobody gets fired for delivering ahead of schedule.

Comment (kmilos) [reply tests/bash_tests/unit_test.py:7]: No biggie, it's clear and specific as is, bet to get that RC out the door.

**Pull request merged**